app/helpers/buyers/applications_helper.rb

```diff
@@ -50,13 +50,13 @@ def last_traffic(cinstance)
     if cinstance.first_daily_traffic_at?
       date = cinstance.first_daily_traffic_at
       title = time_ago_in_words(date) + ' ago'
-      time_tag(date, date.strftime("%B %e, %Y"), :title => title)
+      time_tag(date, date.strftime("%Y-%m-%d"), :title => title)
```

Comment:
is it the same as this?:
```suggestion
      time_tag(date, date.to_s, :title => title)
```
It should be because it is what i see in most of the PR :thinking: in that case, that is simpler